app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,31 @@
+class ReservationCalendar < SimpleDelegator
+
+  attr_accessor :hostname
+
+  def self.to_calendar(reservation)
+    ReservationCalendar.new(reservation).as_ical
+  end
+
+  def initialize(reservation, hostname)
+    super(reservation)
+    @hostname = hostname
+  end
+
+  def as_ical
+    cal = Icalendar::Calendar.new
+    cal.event do |e|
+      e.dtstart = Icalendar::Values::DateTime.new(reserve_start_at)
+      e.dtend = Icalendar::Values::DateTime.new(reserve_end_at)
+      e.summary = "#{facility.abbreviation}: #{product.name}"
+      e.description = "#{facility.name} reservation for #{product.name}. Order number #{order.id}"
```

Comment:
I would probably go with I18n-ing this now. I don't really care if we pass more interpolations than the locale ends up needing, either.

Also, we should use `OrderDetail#to_s`/`OrderDetail#order_number`. That'll give us the full XXX-XXX number including the detail id.